### PR TITLE
[MIG] purchase_all_shipments: Migration to 16.0

### PR DESCRIPTION
**Pull request by augusto-weiss**

### CODE REVIEW

**Review by flotho — APPROVED** [review]
code review
LGTM,

thanks

File: purchase_all_shipments/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        for rec in self:
+            groups = rec.mapped("picking_ids.group_id")
+            all_picking_ids = self.env["stock.picking"].search(
+                [("group_id", "in", groups.ids)]
+            )
+            rec.all_picking_ids = all_picking_ids
+
+    def action_view_all_pickings(self):
+        """Similar to the view_picking method in the purchase module"""
+        action_data = self.env.ref("stock.action_picking_tree_all").read()[0]
```
Comment (vincent-cowboy) [inline purchase_all_shipments/models/purchase_order.py]: This raises `You are not allowed to access 'Action Window' (ir.actions.act_window) records` if the user doesn't belong to `base.group_system`
cf. https://github.com/odoo/odoo/blob/16.0/addons/purchase_stock/models/purchase.py#L140 or https://github.com/OCA/purchase-workflow/pull/1739 to adapt the code

**Review by vincent-cowboy — CHANGES_REQUESTED** [review]
Thank you for the PR. 
I worked on https://github.com/OCA/purchase-workflow/pull/1739 and noticed the action_view_all_pickings method needs to be refactored

**Comment by augusto-weiss** [comment]
Hi @vincent-cowboy !! I made the changes! Could you check that? and please review this one! Thanks a lot

CC: @flotho

**Review by vincent-cowboy — APPROVED** [review]
LGTM, thank you for the PR

**Pull request merged**